.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,94 @@
+name: Bug Report
+description: File a bug report
+title: "[Bug]: "
+labels: ["Triage: Needed"]
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thanks for taking the time to fill out this bug report!
+  - type: textarea
+    id: happened
+    attributes:
+      label: What happened?
+      description: "Tell us what happened. Provide a detailed description of the issue with context."
+    validations:
+      required: true
+  - type: textarea
+    id: expected
+    attributes:
+      label: What you expected?
+      description: "Tell us what you expected to happen."
+    validations:
+      required: true
+  - type: textarea
+    id: possible-fix
+    attributes:
+      label: Possible fix?
+      description: "Tell us about changes that resolve or influence the issue."
+    validations:
+      required: true
+  - type: textarea
+    id: repro
+    attributes:
+      label: Steps to reproduce?
+      description: "Tell us how to reproduce the issue. Walk us through it step by step."
+      value:
+        1.
+        2.
+        3.
+        ...
+    validations:
+      required: true
+  - type: input
+    id: debuglog
+    attributes:
+      label: Debug log
```

Comment:
IMO this is not so good
- you are forcing use pastebin? better hint is use "paste.kodi.tv"
- this field dont allow gran n drop of a direct file, there are often huge logs that pastebin cannot contain

Reply:
I've changed it and reworded.